test/optimus/pr/test_inputs.json

```
@@ -3,7 +3,7 @@
   "TestOptimusPR.reference_matrix": "gs://hca-dcp-sc-pipelines-test-data/optimus-test-data/referenceMatrix.rds",
   "TestOptimusPR.expected_cell_metric_hash": "8bc4a2e96b6404e47f071aa1c06d196f",
   "TestOptimusPR.expected_gene_metric_hash": "c8f0cd1714d08d43bfbd38c6b353b7f9",
-  "TestOptimusPR.expected_loom_file_checksum": "590cf3bf96fb5323adbfad60d635abb2",
+    "TestOptimusPR.expected_loom_file_checksum": "3e4ee79c570cb3e4fc59a26840eaf785",
```

unnecessary whitespace change
```suggestion
   "TestOptimusPR.expected_loom_file_checksum": "3e4ee79c570cb3e4fc59a26840eaf785",
```